x-pack/plugins/cross_cluster_replication/__jest__/client_integration/test_helpers.js

```diff
@@ -0,0 +1,198 @@
+        followerIndexLink.simulate('click');
+      };
+
+      return {
+        selectFollowerIndexAt,
+        openContextMenu,
+        clickContextMenuButtonAt,
+        openTableRowContextMenuAt,
+        clickFollowerIndexAt,
+      };
+    },
+    // Auto-follow patterns user actions
+    autoFollowPatternList() {
+      const { rows } = getMetadataFromEuiTable('ccrAutoFollowPatternListTable');
+
+      const selectAutoFollowPatternAt = (index = 0) => {
+        const row = rows[index];
+        const checkBox = row.reactWrapper.find('input').hostNodes();
+        checkBox.simulate('change', { target: { checked: true } });
+      };
+
+      const clickBulkDeleteButton = () => {
+        find('ccrAutoFollowPatternListBulkDeleteActionButton').simulate('click');
+      };
+
+      const clickConfirmModalDeleteAutoFollowPattern = () => {
+        const modal = find('ccrAutoFollowPatternDeleteConfirmationModal');
+        findTestSubject(modal, 'confirmModalConfirmButton').simulate('click');
+      };
+
+      const clickRowActionButtonAt = (index = 0, action = 'delete') => {
+        const indexLastColumn = rows[index].columns.length - 1;
+        const tableCellActions = rows[index].columns[indexLastColumn].reactWrapper;
+
+        let button;
+        if (action === 'delete') {
+          button = findTestSubject(tableCellActions, 'ccrAutoFollowPatternListDeleteActionButton');
+        } else if (action === 'edit') {
+          findTestSubject(tableCellActions, 'ccrAutoFollowPatternListEditActionButton');
+        }
+
+        if (!button) {
+          throw new Error(`Button for action "${action}" not found.`);
+        }
+
+        button.simulate('click');
+      };
+
+      const clickAutoFollowPatternAt = (index = 0) => {
+        const autoFollowPatternLink = findTestSubject(rows[index].reactWrapper, 'ccrAutoFollowPatternListPatternLink');
+        autoFollowPatternLink.simulate('click');
+      };
+
+      return {
+        selectAutoFollowPatternAt,
+        clickBulkDeleteButton,
+        clickConfirmModalDeleteAutoFollowPattern,
+        clickRowActionButtonAt,
+        clickAutoFollowPatternAt
+      };
+    }
+  };
+
+  return userActions[section]();
+};
+
+export { nextTick, getRandomString, findTestSubject } from '../../../../test_utils';
+
+export const initTestBed = (component, props = {}, options) => {
+  const testBed = registerTestBed(component, {}, ccrStore)(props, options);
+  const getUserActions = initUserActions(testBed);
+
+  return {
+    ...testBed,
+    getUserActions,
+  };
+};
+
+export const mockServerResponses = server => {
+  const mockResponse = (defaultResponse, response) => ([
+    200,
+    { 'Content-Type': 'application/json' },
+    JSON.stringify({ ...defaultResponse, ...response }),
+  ]);
+
+  const mockLoadFollowerIndices = (response) => {
+    const defaultResponse = { indices: [] };
+
+    server.respondWith('GET', /api\/cross_cluster_replication\/follower_indices/,
```

Comment:
Why are these paths written as regular expressions? Can we format them as regular strings so we don't need to escape the slashes?

Reply:
I will check if we need the regex and if not I will use a string 😊